### PR TITLE
open-cluster-management/backlog#8175 Update ICSP

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: install/install_disconnected.adoc [hunk 1]
```diff
@@ -43,22 +43,25 @@ sshKey: >-
 ....
 .. Save the `install-config.yaml` file.
 
-. Deploy a {product-title-short} policy to replace the default registry value with the value of your local registry. For these steps, the policy is named _imageContentSourcePolicy_ for the managed cluster to replace the default registry with the path to the registry instance that you created.
+. Create a yaml file that contains the `ImageContentSourcePolicy` with the name `icsp.yaml` . *Important:* If you modify this on a running cluster, it causes a rolling restart of all nodes.
```
Comment (cdoan1) [inline install/install_disconnected.adoc]: maybe change `icsp.yaml` to `rhacm-policy.yaml`

File: install/install_disconnected.adoc [hunk 2]
```diff
@@ -43,22 +43,25 @@ sshKey: >-
...
-*Note:* Most out-of-the-box policies are functional in a disconnected install, except the image vulnerability policy (`ImageManifestVulnPolicy`).
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: rhacm-repo
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - <mirror-registry>
```
Comment (cdoan1) [inline install/install_disconnected.adoc]: I thought you changed this? If not, ok.

File: install/install_disconnected.adoc [hunk 3]
```diff
@@ -43,22 +43,25 @@ sshKey: >-
...
-... Replace *Unique name* with a name for your policy.
-... Replace *Managed clusters* with the name of the managed cluster that you are updating in the _Cluster binding_ field. *Tip:* You can find this name by viewing the _Cluster details_ page in the {product-title-short} console. 
-... Change the `items.spec.upstream:` value to the path to your local registry. This value points to where your disconnected images are stored.
+. Apply the ImageContentSourcePolicy file by entering the following command:
 +
-See link:../security/create_policy.adoc#managing-security-policies[Managing security policies] for information about creating a policy. 
-.. Select the box for *Enforce if supported*.
-.. Select *Create* to create the policy. 
+----
+oc apply -f icsp.yaml
```
Comment (cdoan1) [inline install/install_disconnected.adoc]: rhacm-policy.yaml

**Review by cdoan1 — APPROVED** [review]

File: install/install_disconnected.adoc (hunk 2 above)
Comment (cadawson) [reply install/install_disconnected.adoc]: I had changed it from mirror-site to mirror-registry. I now changed it back to mirror.registry.com:5000/rhacm2.

**Pull request merged**